modules/administration-guide/partials/proc_setting-up-the-microsoft-azure-devops-services-oauth-app.adoc

```diff
@@ -0,0 +1,48 @@
+:_content-type: PROCEDURE
+:description: Setting up the Microsoft Azure DevOps Services OAuth App
+:keywords: azure, microsoft-azure-devops-services-oauth-app
+:navtitle: Setting up the Microsoft Azure DevOps Services OAuth App
+// :page-aliases:
+
+[id="setting-up-the-microsoft-azure-devops-services-oauth-app"]
+= Setting up the Microsoft Azure DevOps Services OAuth App
+
+Set up a Microsoft Azure DevOps Services OAuth App using OAuth 2.0.
+
+.Prerequisites
+
+* You are logged in to https://azure.microsoft.com/en-us/products/devops/[Microsoft Azure DevOps Services]
+* link:https://www.gnu.org/software/coreutils/base64[`base64`] is installed in the operating system you are using.
+
+.Procedure
+
+. Go to link:https://app.vsaex.visualstudio.com/app/register[].
+
+. Enter the following values:
+
+.. *Company name*: `{prod-short}`.
+.. *Application name*: `{prod-short}`.
+.. *Application website*: `pass:c,a,q[{prod-url}]/`
+.. *Authorization callback URL*: `pass:c,a,q[{prod-url}]/api/oauth/callback`
+
+. *Select Authorized scopes*: `Code (read and write)`.
+
+. Click *Create application*.
+
+. Copy the *App ID* and encode it to Base64 for use when applying the Microsoft Azure DevOps Services OAuth App Secret:
++
+[subs="+quotes,+attributes,+macros"]
+----
+$ echo -n '__<microsoft_azure_devops_services_oauth_app_id>__' | base64
+----
+
+. Click *Show* next to Client Secret and Copy the *Client Secret* and encode it to Base64 for use when applying the Microsoft Azure DevOps Services OAuth App Secret:
```

Comment:
@tolusha, when you have a minute, could you please take and share a screenshot of this area of the [page](https://app.vsaex.visualstudio.com/app/register), if possible?

Reply:
I'd also like to double-check with you whether the UI on those pages names it `App ID` or `Application ID` when the value is created and available to the user.